FWCore/Services/plugins/InitRootHandlers.cc

```diff
@@ -164,12 +165,14 @@ namespace {
        "nbins is <=0 - set to nbins = 1",
        "nbinsy is <=0 - set to nbinsy = 1"}};
 
-  constexpr std::array<const char* const, 6> in_location{{"Fit",
+  //Location generating messages which should be reported as an INFO not a ERROR
+  constexpr std::array<const char* const, 7> in_location{{"Fit",
                                                           "TDecompChol::Solve",
                                                           "THistPainter::PaintInit",
                                                           "TUnixSystem::SetDisplay",
                                                           "TGClient::GetFontByName",
-                                                          "Inverter::Dinv"}};
+                                                          "Inverter::Dinv",
+                                                          "RTaskArenaWrapper"}};
```

Comment:
```suggestion
                                                          "RTaskArenaWrapper",
                                                          "TThreadExecutor::ParallelReduce",
                                                          "TThreadExecutor::ParallelFor"}};
```

Reply:
size of array also needs an update ( https://github.com/cms-sw/cmssw/pull/32782/files#diff-557c0474dee6f7cb096e823325eb929670d9fe68e9ced2ff73fac38bd449e6c5R169 )